frontend/survey-project/src/app/components/pages/register-page/register-page.component.html

```diff
@@ -44,46 +32,29 @@ <h2 class="create-account-text">Create a new account</h2>
 
     <div class="row mb-4">
       <div class="col-md-4 col-12">
-        <app-input
-          label="First Name"
-          type="text"
-          name="name"
-          placeholder="Enter your first name"
-          [required]="true"
+        <app-input label="First Name" type="text" name="name" placeholder="Enter your first name" [required]="true"
           formControlName="name"
-          [hasError]="(registerForm.get('name')?.invalid && registerForm.get('name')?.touched) ?? false"
-        ></app-input>
+          [hasError]="(registerForm.get('name')?.invalid && registerForm.get('name')?.touched) ?? false"></app-input>
         <div *ngIf="registerForm.get('name')?.invalid && registerForm.get('name')?.touched" class="text-danger">
           <small *ngIf="registerForm.get('name')?.errors?.['required']">First name is required.</small>
           <small *ngIf="registerForm.get('name')?.errors?.['maxlength']">Maximum 100 characters allowed.</small>
         </div>
       </div>
       <div class="col-md-4 col-12 mt-3 mt-md-0">
-        <app-input
-          label="Last Name"
-          type="text"
-          name="lastname"
-          placeholder="Enter your last name"
-          [required]="true"
+        <app-input label="Last Name" type="text" name="lastname" placeholder="Enter your last name" [required]="true"
           formControlName="lastname"
-          [hasError]="(registerForm.get('lastname')?.invalid && registerForm.get('lastname')?.touched) ?? false"
-        ></app-input>
+          [hasError]="(registerForm.get('lastname')?.invalid && registerForm.get('lastname')?.touched) ?? false"></app-input>
         <div *ngIf="registerForm.get('lastname')?.invalid && registerForm.get('lastname')?.touched" class="text-danger">
           <small *ngIf="registerForm.get('lastname')?.errors?.['required']">Last name is required.</small>
           <small *ngIf="registerForm.get('lastname')?.errors?.['maxlength']">Maximum 100 characters allowed.</small>
         </div>
       </div>
       <div class="col-md-4 col-12 mt-3 mt-md-0">
-        <app-input
-          label="Phone Number"
-          type="tel"
-          name="phoneNumber"
-          placeholder="Enter your phone number"
-          [required]="true"
-          formControlName="phoneNumber"
-          [hasError]="(registerForm.get('phoneNumber')?.invalid && registerForm.get('phoneNumber')?.touched) ?? false"
-        ></app-input>
-        <div *ngIf="registerForm.get('phoneNumber')?.invalid && registerForm.get('phoneNumber')?.touched" class="text-danger">
+        <app-input label="Phone Number" type="tel" name="phoneNumber" placeholder="Enter your phone number"
```

Comment:
When we have a lot of fields in one component, we should leave it and put it to be every field in a new row.
This should apply to all changes in this PR.